### PR TITLE
feat: add image import

### PR DESCRIPTION
**Pull request by tjosepo**
Hi! I'm interested in Packup and would love to see this project grow, so I made this PR. 

**Changes:**
- When using `serve` or `build`, an image declared inside a of a ```` element will be imported.
- Added an image import to `examples/with-simple-assets/index.html` and `examples/with-imports/index.html`.
- Added tests.

**Example:**
```diff
<html>
  <head>
  </head>
  <body>
    <h1>Hey!</h1>
-   <img src="images/my_image.png">
+   <img src="index.e64db08d87e7ea59f8a01df1dcbb9de5.png">
</body></html>
```
A new image file is created. The orignal image `src` will be replaced by the new image filename.

**Todos:**
- Documentation
- Add a file-loader plugin to Esbuild to be able to import images from JavaScript (`import url from './image.png";`).

Feedback is welcomed 🙂

### CODE REVIEW

**Comment by tjosepo** [comment]
By the way, the image I used for testing is the Deno logo.

**Comment by tjosepo** [comment]
Just realized I forgot about the `srcset` attribute and the `<picture>` element. I'm considering adding them in a separate PR.

File: generate_assets.ts [hunk 1]
```diff
@@ -289,11 +290,55 @@ class ScriptAsset implements Asset {
...
+class ImageAsset implements Asset {
+  static create(img: Element): ImageAsset | null {
+    const src = img.getAttribute("src");
+    if (!src) {
+      logger.warn(
+        "<img> tag doesn't have src attribute",
+      );
+      return null;
+    }
+    if (!src || src.startsWith("http://") || src.startsWith("https://")) {
```
Comment (kt3k) [inline generate_assets.ts:303]: nice!

**Review by kt3k — APPROVED** [review]
Thank you for doing this work!

Nicely implementd, nicely tested, examples works, perfect! LGTM

**Comment by kt3k** [comment]
published as 0.1.11

**Pull request merged**